uber/templates/regform.html

```diff
@@ -295,11 +295,17 @@ <h4>{{ tier.name }}</h4>
 
                 <p class="help-block col-sm-offset-3 col-sm-9">
                     Each level includes all lower levels. <br/>
-                    Supporter level and higher {% if c.BEFORE_SUPPORTER_DEADLINE %}are{% else %}were{% endif %} only available until {{ c.SUPPORTER_DEADLINE|datetime_local }}.
+                    Supporter level and higher {% if c.BEFORE_SUPPORTER_DEADLINE %}are{% else %}were{% endif %} only available until {{ c.SUPPORTER_DEADLINE|datetime_local }}.<br/>
+                    Shirt level up to Supporter level {% if c.BEFORE_SHIRT_DEADLINE %}are{% else %}were{% endif %} only available until {{ c.SHIRT_DEADLINE|datetime_local }}.
                 </p>
             </div>
         {% endif %}
     {% endif %}
+  {% if (not c.PREREG_DONATION_OPTS or c.PREREG_DONATION_OPTS|length <= 1) and c.PAGE_PATH in ['/preregistration/form', '/preregistration/post_form', '/preregistration/register_group_member'] %}
```

Comment:
This is restricted to the new prereg pages for two reasons:
1. If an attendee has a kick-in already, this would be confusing, especially since it would come right after the kick-in box showing them the tier they selected.
2. There's never been an expectation of kick-ins existing for at-door registrations. I don't think. Anyway, the help text for "Extra Donation" should be clear enough.

Reply:
Famous last words.